analysis_driver/dataset.py

```diff
@@ -498,10 +498,10 @@ def _pipeline_instruction(self):
         return instruction
 
     def _is_ready(self):
-        return self.data_threshold and int(self._amount_data()) > int(self.data_threshold)
+        return self.data_threshold and self.pc_q30 > 75 and int(self._amount_data()) > int(self.data_threshold)
 
     def report(self):
-        runs = sorted(set(r.get(ELEMENT_RUN_NAME) for r in self.run_elements))
+        runs = self.sample.get('run_ids')
```

Comment:
`run_id` is not part of  `sample`. It should be `sample.get('aggregated').get('run_ids')`